### ISSUE TITLE
Feature request: Trading

### ISSUE THREAD

**Issue by cstevens780** [110yards/fantasy#18]
Would be nice to have trading available on the website.

**Comment by argos1873** [comment]
was wondering what the status of this is... has trading been added as a feature, and I'm just missing something?

**Comment by mdryden** [comment]
No, this feature didn't make it in for 2022.  For trades I recommend you have two managers just coordinate add/drops, ideally while communicating to the rest of the league what's happening.

This is high on the list though and might get implemented sometime during the season.
